src/main/java/com/Project/accountManager/entities/Account.java

```diff
@@ -0,0 +1,31 @@
+package com.Project.accountManager.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+@Entity
+@Table(name="account")
+@Data
+public class Account {
+    @Id
+    private Long id;
+    int money;
+    int accountNumber;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JsonIgnore
+    User user;
+
+    public void setId(Long id) {
+        this.id = id;
```

Comment:
@Data annotation is already contains getter and setter.
Either delete setters and only use @Data annotation or delete @data annotation and use setters manually

Reply:
OK, necessary corrections will be made.